package.json

```diff
@@ -5,13 +5,16 @@
   "scripts": {
     "postinstall": "npm run postinstall:electron && electron-builder install-app-deps",
     "postinstall:electron": "node postinstall",
+    "postinstall:web": "node postinstall-web",
     "ng": "ng",
     "start": "npm run postinstall:electron && npm-run-all -p codegen:gql-watch ng:serve electron:serve",
     "build": "npm run postinstall:electron && npm run codegen:gql && npm run electron:serve-tsc && ng build",
-    "build:dev": "npm run codegen:gql && npm run build -- -c dev",
-    "build:prod": "npm run codegen:gql && npm run build -- -c production",
+    "build:dev": "npm run build -- -c dev",
+    "build:prod": "npm run build -- -c production --base-href=./",
+    "build:prod:web": "npm run codegen:gql && ng build --prod --base-href=https://CATcher-org.github.io/CATcher/",
     "ng:serve": "ng serve",
     "ng:serve:web": "npm run postinstall:web && ng serve -o",
+    "deploy:web": "npm run build:prod:web && ngh --dir=dist --name=CATcher --email=kjunwei1996@gmail.com --no-silent",
```

Comment:
Just a small comment: Will we be using your email to do the web deployment from here onwards? :O

Reply:
Having the email there will tag the commit on the `gh-pages` branch with stated email. 

@damithc, maybe we should use your email instead?

Reply:
How about `se-edu@comp.nus.edu.sg`? I receive the emails sent to that address.

Reply:
Alright changed! 👍